src/Components/Quizz/Modal.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import './Modal.css';
+import PalpatineSon1 from './sound/palpatine.mp3'
+import YodaSon1 from './sound/Yoda.mp3'
+
+const modal = (props) => {
+
+    return (
+        <div className='ContentPicModalOff'>
+            <p className='contentModalP'>
+            Level 1
+            </p>
+            <div className="modal-wrapper"
+                style={{
+                    transform: props.show ? 'translateY(0vh)' : 'translateY(-100vh)',
+                    opacity: props.show ? '1' : '0'
+                }}>
```

Comment:
Use conditional classNames instead of conditional inline styles. Most of the time you don't need to write inline styles unless you want to animate using javascript a specific css property.